File: Chapter 3/TestNumber.cs
using System;

namespace TestNumber
{
    class Program
    {
        public static void Main()
        {

            Number number1 = new Number();
            Number number2 = new Number();
            Number number3 = new Number();

        }
    }

    public class Number
    {

        int count = 0;	// available to both method1 and method2 

        public void method1()
        {
            count = 1;
        }

        public void method2()
        {
            count = 2;
        }

    }
}

File: Chapter 3/TestNumber.java
public class TestNumber {
	
	public static void main(String[] args) {

		Number number1 = new Number();
		Number number2 = new Number();
		Number number3 = new Number();
		
	}
}
class Number {

    	
	int count;	// available to both method1 and method2 

    	public void method1() {
        	count = 1;
    	}

    	public void method2() {
        	count = 2;
    	}

}